docs/podman-system-renumber.1.md

```diff
@@ -1,7 +1,7 @@
 % podman-system-renumber(1)
 
 ## NAME
-podman\-system\-renumber - Renumber container locks
+podman\-system\-renumber - Migrate lock numbers to handle a change in maximum number of locks
```

Comment:
Migrate isn't a good word here, it brings up a bad connotation.  Suggest: "Acquire container locks to handle a change in the maximum number of locks"